addons/medical/functions/fnc_setUnconscious.sqf

```diff
@@ -46,6 +46,14 @@ if (currentWeapon _unit != primaryWeapon _unit) then {
     _unit selectWeapon primaryWeapon _unit;
 };
 
+if (QGVAR(dropWeaponOnUnconscious)) then {
+    if (_unit == ACE_player) then {
+        if (_knockOut) then {
+            player call EFUNC(hitreactions,throwWeapon);
```

Comment:
Why should the Zeus throw their weapon when the remote controlled AI gets shot?

Reply:
Hehe, they should not.

Assuming that this should be suffisant.
```suggestion
            ACE_player call EFUNC(hitreactions,throwWeapon);
```
 also dont want ai to drop their weapons cause ai